changelogs/unreleased/gh-11115-reconnect-timeout.md

```diff
@@ -0,0 +1,5 @@
+## feature/config
+
+* Added the `replication.reconnect_timeout` option, which shows
+  the timeout (in seconds) between attempts to reconnect to a
+  master in case of connection failure (gh-11115, part of gh-10059).
```

Comment:
From the documentation request:

> Since 3.3.2.

It means that we're going to backport a feature to a patch level release. Are we really going to do so or is the release version incorrect here?

Reply:
This is error, should be `3.4.0`. Fixed